scripts/make_docs_locally.sh

```diff
@@ -4,16 +4,16 @@
 # python scripts/json_schema_to_md_doc.py
 # python scripts/generate_dialect_comparison_docs.py
 
-# NB not including docs/demos, as 02_Exploratory_analysis.ipynb may
-# cause process to hang on: Running 3 `page_markdown` events
-# only an issue with `serve` or `build` - `gh-deploy` works okay
+# and preprocess include files with python scripts/preprocess.py
+# as include-markdown plugin clashes with mknotebooks
+# make sure not to commit changes to files with inclusions
 
-# UPDATE="TRUE"
-# if [[ $UPDATE == "TRUE" ]]
-# then
-#     rm -rf docs/demos/
-#     git clone https://github.com/moj-analytical-services/splink_demos.git docs/demos/
-# fi
+UPDATE="TRUE"
+if [[ $UPDATE == "TRUE" ]]
+then
+    rm -rf docs/demos/
+    git clone https://github.com/moj-analytical-services/splink_demos.git docs/demos/
+fi
 
 deactivate
 python3 -m venv docs-venv
```

Comment:
Do we want to add `python3 scripts/preprocess_markdown_includes.py` to this script?

Reply:
This is a good point - oversight on my part.